client/js/components/ParkFeatureListItem.jsx

```diff
@@ -4,18 +4,23 @@ import icons from '../utils/icons.json';
 
 
 export default class ParkFeatureListItem extends React.Component {
+    onClick() {
+        this.props.selectFeature(this.props.id);
+        this.props.slideUp();
```

Comment:
I think the call to `this.props.slideUp()` should live in `selectFeature()`.

That way `ParkFeatureListItem` doesn't need to know about `slideUp()`.